### PR TITLE
[feat]: Adding css for inputs

### PR DESCRIPTION
**Pull request by raghav2110**
Hi, I have added css for input field. Please find SS below and review my PR.

Screenshot 2022-01-30 at 9 34 31 PM

### CODE REVIEW

File: components/inputs/inputs.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    border: 1px solid #F44336;
+}
+
+.input-group.err input:focus{
+    border: 1px solid #F44336;
+}
+
+.error-message{
+    color: #F44336;
+}
```
Comment (ritik-patel05) [inline components/inputs/inputs.css]: Add a new line at the end of the file.
Reference: [link](https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log)

**Review by ritik-patel05 — COMMENTED** [review]
The code looks good.

can add Red color as danger and use it in every component and website. It will help you maintain consistency for viewers, and can change the red color to some other shade in the future if you want to.

File: components/inputs/inputs.css (hunk 1 above)
Comment (raghav2110) [reply components/inputs/inputs.css]: Got it will keep this in mind from next PR

File: components/inputs/inputs.css [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    margin: 0 1.2rem 0.8rem;
+    width: 20%;
+}
+  
+label {
+    display: block;
+    margin-bottom: 2px;
+}
+
+input[type=text] {
```
Comment (prachi-sahani) [inline components/inputs/inputs.css]: doesn't this restrict the usage of input to only type text for the user? what if, they want the input of some other type, will the styles still get applied?

**Review by prachi-sahani — COMMENTED** [review]
Apart from the mentioned point, the code seems fine. 👍🏻 
Forgot to mention: I feel that the font size of the error should be little smaller than the label's font size or vice-versa

File: components/inputs/inputs.css (hunk 2 above)
Comment (Hamdrive) [reply components/inputs/inputs.css]: > doesn't this restrict the usage of input to only type text for the user? what if, they want the input of some other type, will the styles still get applied?

using `text` property should account for both alphanumeric and symbols, hence it covers all possible text input by user

Comment (raghav2110) [reply components/inputs/inputs.css]: Yes it was on purpose written like that.

Comment (prachi-sahani) [reply components/inputs/inputs.css]: > > doesn't this restrict the usage of input to only type text for the user? what if, they want the input of some other type, will the styles still get applied?
> 
> using `text` property should account for both alphanumeric and symbols, hence it covers all possible text input by user

yes it does cover alphanumeric values and numbers but the styles won't apply if the user(user of this component library) sets the type other than "text"(for eg: number, email, etc.)

Comment (raghav2110) [reply components/inputs/inputs.css]: Got your point prachi, thanks for bringing that up. Will try another approach for it.

**Pull request merged**